server/utils/crypto.ts

```diff
@@ -17,3 +19,54 @@ export function safeEqual(a?: string, b?: string) {
 
   return crypto.timingSafeEqual(Buffer.from(a), Buffer.from(b));
 }
+
+const formatter: CryptoJS.lib.CipherParams["formatter"] = {
```

Comment:
I don't want to introduce new encrypt/decrypt methods, can we lose all this? It's unneccessary